src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -211,6 +212,8 @@
     private String packageName = "";
 
     private String packagePath = "";
+    
+    private boolean usingSandbox = false;
```

Comment:
remove usingSandbox field.   More below on moving the code that
sets usingSandbox into a new method.

Reply:
rather than using a field, use a local variable in executeProgram for the two places where this field variable is used.

Reply:
done.